resource-managers/kubernetes/integration-tests/README.md

```diff
@@ -17,9 +17,9 @@ To run tests with Java 11 instead of Java 8, use `--java-image-tag` to specify t
 
     ./dev/dev-run-integration-tests.sh --java-image-tag 11-jre-slim
 
-To run tests with Hadoop 2.7 instead of Hadoop 3.2, use `--hadoop-profile`.
+To run tests with Hadoop 2 instead of Hadoop 3, use `--hadoop-profile`.
```

Comment:
nit: maybe `To run tests with Hadoop 2.x instead of Hadoop 3.x`

Reply:
Done